core/templates/dev/head/domain/feedback_thread/FeedbackThreadSummaryObjectFactory.js

```diff
@@ -36,48 +36,49 @@ oppia.factory('FeedbackThreadObjectFactory', [function() {
     this.threadId = threadId;
   };
 
-  FeedbackThread.prototype.updateSummaryOnNewMessage = function(
+  FeedbackThread.prototype.markTheLastTwoMessagesAsRead = function() {
+    if (this.authorSecondLastMessage) {
+      this.secondLastMessageRead = true;
+    }
+    this.lastMessageRead = true;
+  }
+
+  FeedbackThread.prototype.updateThreadSummaryOnAdditionOfNewMessage = function(
```

Comment:
This ends up being called as threadSummary.updateThreadSummary... which is a little weird. Just say threadSummary.appendNewMessage() or similar.